lib/SPIRV/LLVMToSPIRVDbgTran.cpp

```diff
@@ -490,7 +490,10 @@ LLVMToSPIRVDbgTran::transDbgCompilationUnit(const DICompileUnit *CU) {
   Ops[SPIRVDebugInfoVersionIdx] = SPIRVDebug::DebugInfoVersion;
   Ops[DWARFVersionIdx] = M->getDwarfVersion();
   Ops[SourceIdx] = getSource(CU)->getId();
-  Ops[LanguageIdx] = CU->getSourceLanguage();
+  Ops[LanguageIdx] = spv::SourceLanguage::SourceLanguageUnknown;
```

Comment:
Ops[LanguageIdx] is assigned to twice. Shouldn't the first assignment to it be removed, now?

Reply:
Thanks for catching that! Fixed